### PR TITLE
[tizen_app_control] Change the example app language to C#

### PR DESCRIPTION
**Pull request by JSUYA**
Now C# multi type application is supported, so updated it.

### CODE REVIEW

File: packages/tizen_app_control/example/tizen/service/tizen-manifest.xml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="org.tizen.tizen_app_control_example" version="1.0.0" api-version="5.5" xmlns="http://tizen.org/ns/packages">
-    <profile name="wearable"/>
-    <service-application appid="org.tizen.tizen_app_control_example_service" exec="runner_service" type="capp" multiple="false" nodisplay="true" taskmanage="false" auto-restart="false">
+<manifest package="com.tizen.tizen_app_control_example" version="1.0.0" api-version="5.5" xmlns="http://tizen.org/ns/packages">
```
Comment (swift-kim) [inline packages/tizen_app_control/example/tizen/service/tizen-manifest.xml:2]: Nope

File: packages/tizen_app_control/example/tizen/ui/tizen-manifest.xml [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+    <ui-application appid="org.tizen.tizen_app_control_example" exec="Runner.dll" type="dotnet" multiple="false" nodisplay="false" taskmanage="true">
         <label>tizen_app_control_example</label>
         <icon>ic_launcher.png</icon>
+        <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true"/>
     </ui-application>
     <privileges>
         <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
         <privilege>http://tizen.org/privilege/appmanager.kill.bgapp</privilege>
+        <privilege>http://tizen.org/privilege/call</privilege>
+        <privilege>http://tizen.org/privilege/download</privilege>
```
Comment (swift-kim) [inline packages/tizen_app_control/example/tizen/ui/tizen-manifest.xml:13]: Forgot to remove?

**Review by xiaowei-guan — APPROVED** [review]

File: packages/tizen_app_control/example/lib/main.dart [hunk 3]
```diff
@@ -185,12 +185,12 @@ class _MyAppState extends State<MyApp> {
             children: <Widget>[
               ElevatedButton(
                 onPressed: _sendSms,
-                child: const Text('Send SMS'),
+                child: const Text('Send SMS(No TV)'),
```
Comment (swift-kim) [inline packages/tizen_app_control/example/lib/main.dart:188]: Can we find a better example for TV? (as we are deprecating the wearable profile)

File: packages/tizen_app_control/example/tizen/service/tizen-manifest.xml (hunk 1 above)
Comment (JSUYA) [reply packages/tizen_app_control/example/tizen/service/tizen-manifest.xml]: my mistake

File: packages/tizen_app_control/example/tizen/ui/tizen-manifest.xml (hunk 2 above)
Comment (JSUYA) [reply packages/tizen_app_control/example/tizen/ui/tizen-manifest.xml]: done

File: packages/tizen_app_control/example/lib/main.dart (hunk 3 above)
Comment (JSUYA) [reply packages/tizen_app_control/example/lib/main.dart]: I removed that did not work. And I found 'web page open' is works. so i added it.

**Review by swift-kim — APPROVED** [review]
Thanks!

**Pull request merged**